sklearn/neighbors/tests/test_neighbors.py

```diff
@@ -108,6 +109,22 @@ def test_unsupervised_inputs():
         assert_array_almost_equal(ind1, ind2)
 
 
+def test_n_neighbors_datatype():
+    # Test to check whether n_neighbors is integer
+    x = [[1, 1], [1, 1], [1, 1]]
+    n = 3.
```

Comment:
please avoid single letter variable, except `X`and `y`.
What about `n_neighbors`?